[2021-08-12 20:51:07,947] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: pipeline_try.thursday_Task 2021-08-12T20:50:54.723979+00:00 [queued]>
[2021-08-12 20:51:07,983] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: pipeline_try.thursday_Task 2021-08-12T20:50:54.723979+00:00 [queued]>
[2021-08-12 20:51:07,986] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-08-12 20:51:07,987] {taskinstance.py:1088} INFO - Starting attempt 1 of 2
[2021-08-12 20:51:07,988] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-08-12 20:51:08,007] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): thursday_Task> on 2021-08-12T20:50:54.723979+00:00
[2021-08-12 20:51:08,015] {standard_task_runner.py:52} INFO - Started process 32721 to run task
[2021-08-12 20:51:08,022] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'pipeline_try', 'thursday_Task', '2021-08-12T20:50:54.723979+00:00', '--job-id', '108', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_try1.py', '--cfg-path', '/tmp/tmpz_ey4skj', '--error-file', '/tmp/tmpoejiftf6']
[2021-08-12 20:51:08,030] {standard_task_runner.py:77} INFO - Job 108: Subtask thursday_Task
[2021-08-12 20:51:08,159] {logging_mixin.py:104} INFO - Running <TaskInstance: pipeline_try.thursday_Task 2021-08-12T20:50:54.723979+00:00 [running]> on host 99300daf4d26
[2021-08-12 20:51:08,331] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=ameysonje242000@gmail.com
AIRFLOW_CTX_DAG_OWNER=amey
AIRFLOW_CTX_DAG_ID=pipeline_try
AIRFLOW_CTX_TASK_ID=thursday_Task
AIRFLOW_CTX_EXECUTION_DATE=2021-08-12T20:50:54.723979+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-08-12T20:50:54.723979+00:00
[2021-08-12 20:51:08,347] {logging_mixin.py:104} INFO - From thursday_Task, so its thursday today
[2021-08-12 20:51:08,350] {python.py:151} INFO - Done. Returned value was: None
[2021-08-12 20:51:08,374] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=pipeline_try, task_id=thursday_Task, execution_date=20210812T205054, start_date=20210812T205107, end_date=20210812T205108
[2021-08-12 20:51:08,455] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-08-12 20:51:08,490] {local_task_job.py:149} INFO - Task exited with return code 0
